components/landing_page.module.css

```diff
@@ -0,0 +1,135 @@
+/* Wrapper for landing page content */
+.body {
+	margin-right: auto;
+	margin-left: auto;
+	max-width: 1440px;
+	padding-right: 100px;
+	padding-left: 100px;	
+}
+
+/* General Column */
+.column {
+	display: flex;
+	flex-direction: column;
+	flex-basis: 100%;
+	flex: 1;
+	justify-content: center;
+	align-items: center;
+  padding-right: 30px;
+  padding-left: 30px;
+  padding-top: 30px;
+  padding-bottom: 30px;
+}
+
+/* General Row */
+.row {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  max-width: 100%;
+  justify-content: center;
+} 
+
+/* Main Text */
+.TCFG {
+  width: 715px;
+  height: 184px;
+  font-family: GujaratiSangamMN;
+  font-size: 72px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: center;
+  color: #000000;
+  padding-bottom: 30px;
+}
+
+/* Description text */
+.text {
+  font-size: 20px;
+  min-width: 300px;
+  padding-bottom: 40px;
+  font-family: HiraginoSans-W1;
```

Comment:
<img width="767" alt="Screen Shot 2020-07-14 at 12 23 50 PM" src="https://user-images.githubusercontent.com/67434940/87450770-f3a68c00-c5cc-11ea-91a4-c40aaab8baef.png">

`HiraginoSans-W1` is non-standard is and is not available in browsers. In order to make it available, it needs to be manually loaded, either in the root `index.html` file or or the global css file. That can definitely be done in a future PR, but I wanted to point it out here because the font is showing up for me currently as Times New Roman which is a serif font!

A very common approach for handling font load errors is to provide backup fonts that are 1) more standard and 2) shipped with operating systems/browsers so they don't need to be loaded. You can provide as many backup fonts as you want, and it is generally good to end this list with the _kind_ of font, like `serif` or `sans-serif` or `monospace`. If it defaults to the kind, then the browser will choose the default system font for that kind. In most cases, this isn't ideal, but it is much better to show any `sans-serif` font if that is desired than a `serif` font.

```suggestion
  font-family: "HiraginoSans-W1", sans-serif;
```

Reply:
Very helpful -- I didn't know how the downloading of fonts on browsers worked. It seems that we have to download a .ttf/.otf file of the font to include in our index.html page, but I'm not sure where we can find that file this specific font. Is this one we would have to pay for?